src/vs/workbench/parts/files/electron-browser/fileCommands.ts

```diff
@@ -78,7 +79,10 @@ export const REMOVE_ROOT_FOLDER_LABEL = nls.localize('removeFolderFromWorkspace'
 
 export const openWindowCommand = (accessor: ServicesAccessor, paths: string[], forceNewWindow: boolean) => {
 	const windowsService = accessor.get(IWindowsService);
-	windowsService.openWindow(paths, { forceNewWindow });
+	const currentWindow = remote.getCurrentWindow();
+	const targetWindowId = currentWindow && currentWindow.id;
+
+	windowsService.openWindow(paths, { forceNewWindow, targetWindowId });
 };
 
 function save(resource: URI, isSaveAs: boolean, editorService: IWorkbenchEditorService, fileService: IFileService, untitledEditorService: IUntitledEditorService,
```

Comment:
Getting the current window id here and passing it down to the `openWindow` method.